### ISSUE TITLE
some way to use this library with node compiling ?

### ISSUE THREAD

**Issue by hayden-t** [perliedman/geojson-path-finder#52]
Im a bit new to node, and it took me  while to work out with your library that though not stated i (think) i had to use browserify to 'compile' the basics of your library to get it to work in the browser ?

Is there any way to use the library without this, eg just <script> tags includes ?

**Comment by perliedman** [comment]
No, since it's built on modules (CommonJS), you have to bundle it to use in a browser.

Having a way to easily create a bundle is a good idea and something at least one person asked about before. I would happily accept a PR for this, but given that I don't actively work at this project, it's not likely I will add it anytime soon.

**Comment by nickw1** [comment]
I had to do this myself as I wanted to use GeoJSON Path Finder with Pannellum and encountered issues when combining them and using browserify.

Having little time to diagnose the issues, I decided to create a 'de-noded' version of GeoJSON Path Finder a few months back - note it's the 'browser' branch. Here is the repo if you think it might be of use.

https://github.com/nickw1/geojson-path-finder/tree/browser

**Comment by psociety** [comment]
@hayden-t probably late, but yes, there is:
```html
<script src="https://bundle.run/geojson-path-finder@1.5.3"></script>
```

https://jsfiddle.net/6qugy0rm/

**Comment by perliedman** [comment]
Since a lot has happened with module support in browsers and generally how this project is built and bundled, I don't think this is relevant any more, closing.
